lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -663,22 +633,32 @@ def transaction(&block)
     # Returns true if the user has the operation on the object
     def is_privilege?(user_or_attribute, operation, object_or_attribute)
       policy_classes_containing_object = policy_classes_for_object_attribute(object_or_attribute)
+      operation_id = operation.try(:unique_identifier) || operation.to_s
+
       if policy_classes_containing_object.count < 2
-        is_privilege_single_policy_class(user_or_attribute, operation, object_or_attribute)
+        accessible_operations(user_or_attribute, object_or_attribute, operation_id).any?
```

Comment:
I know it's not pretty, but since this is gonna be our bread-and-butter method, maybe 
`!accessible_operations(user_or_attribute, object_or_attribute, operation_id).empty?`

`!a.empty?` is significantly faster than `a.any?`, according to Googlefu